controllers/controllers.js

```diff
@@ -20,3 +24,9 @@ exports.viewReviews = (req, res, next) => {
       next(err);
     });
 };
+
+exports.viewUsers = (req, res) => {
+  selectUsers().then(user => {
+    res.status(200).send({ users: user });
+  });
```

Comment:
we want to have a catch block if we are going to manually be throwing errors and passing them to an error handler